multiapps-controller-persistence/src/main/java/org/cloudfoundry/multiapps/controller/persistence/services/FileStorage.java

```diff
@@ -22,4 +22,6 @@ public interface FileStorage {
 
     <T> T processFileContent(String space, String id, FileContentProcessor<T> fileContentProcessor) throws FileStorageException;
 
+    void testConnection();
```

Comment:
I think it will be good to mark with comment or throws that this method can throw exception in case of error.